internal/cmd/local/command_kafka_start.go

```diff
@@ -58,31 +61,33 @@ func (c *Command) kafkaStart(cmd *cobra.Command, args []string) error {
 	}
 	defer out.Close()
 
-	buf := new(strings.Builder)
-	_, err = io.Copy(buf, out)
-	if err != nil {
-		return err
-	}
+	scanner := bufio.NewScanner(out)
+	for scanner.Scan() {
+		var response ImagePullResponse
```

Comment:
This way, we don't need to use the `&` character whenever we pass this into a function.
```suggestion
		response := new(ImagePullResponse)
```